source/server/spongineer/commands.rst

```diff
@@ -1,6 +1,6 @@
-========
-Commands
-========
+===============
+Ingame Commands
```

Comment:
why? commands can be used from console too and not just in game, honestly if anything this page should be called "Commands and Permissions"

Reply:
I first though about calling it `Server Commands` to separate it from `Plugin Commands`.